src/Silakov.Homework4/Task.cs

```diff
@@ -32,9 +32,20 @@ internal class Task
         /// </summary>
         private DateTime _executionDate;
 
+        /// <summary>
+        /// Счётчик.
+        /// </summary>
         private static int counter = 1;
 
-        Task()
+        public int Id 
```

Comment:
Can be written as:
public int Id { get; }
or
public int Id { get;  private set; }
C# can do a lot of things for us